user_interactions/user_interaction_txt_text.py

```diff
@@ -0,0 +1,69 @@
+import os
+from datetime import datetime
+
+from user_interactions.base_user_interaction import (BaseUserInteraction,
+                                                     MessageScope)
+
+
+class UserInteractionFileManager():
```

Comment:
For discussion:
By putting this class to in user_interaction_txt_text module you've showed it's connection to UserInteraction.
Don't think, you should represent this connection in name too.

Reply:
Also, not sure that ...txt_text is a appropriate name for module

Reply:
> Also, not sure that ...txt_text is a appropriate name for module

Let's live it as it is, and fix it later, if we have a need.

Reply:
> For discussion:
> By putting this class to in user_interaction_txt_text module you've showed it's connection to UserInteraction.
> Don't think, you should represent this connection in name too.

Name was chanhed.